modules/ROOT/pages/_partials/mmp-deploy-to-cloudhub-2.adoc

```diff
@@ -119,6 +119,8 @@ If not set, defaults to +https://anypoint.mulesoft.com+. | No
 Ensure that this value is equal to or higher than the earliest required Mule version of your application. +
 Example values: `4.6.0` selects the last build where it defaults to Edge channel and Java 8. If you want to select a different channel or Java version, specify the full Mule version name as in `4.6.0:1e-java17`. +
 When using Mule Maven plugin versions 3.8.0 and 4.0.0 for deployments, the `muleVersion` property doesn't allow you to specify the release channel and Java version which may cause unexpected behavior. To specify the release channel and Java version, upgrade the plugin to version 4.1.0. See xref:release-notes::mule-maven-plugin/mule-maven-plugin-4.1.0-release-notes.adoc[Mule Maven Plugin 4.1.0 Release Notes].  | Yes
+| `releaseChannel`| Set the name of the release channel to use for the selected Mule version. Supported values are `NONE`, `EDGE`, and `LTS`. If you don't specify a Mule version, the default Mule version for the selected release channel is used. If the selected release channel doesn't exist, an error occurs. | No
+| `javaVersion` | Set the name of the Java version to use for the selected Mule version. Supported values are `8` and `17`. If you don't specify a Mule version, the default Mule version for the selected Java version is used. If the Java version you select isn't available for the specified Mule version, an error occurs. | No
```

Comment:
Could we indicate the default values? 🤔 
According to the API documentation:
- releaseChannel : EDGE
- javaVersion: 8

https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/amc-application-manager/minor/4.0/console/type/%23148/